cabal.project

```diff
@@ -51,6 +51,7 @@ write-ghc-environment-files: never
 index-state: 2022-08-15T06:53:13Z
 
 constraints:
+  entropy == 0.4.1.10,
```

Comment:
please add a comment explaining why this is here and when we can remove it

Reply:
I've relaxed it to a lower bound and added a comment.